tree/ntuple/v7/test/UnsplitLinkDef.h

```diff
@@ -0,0 +1,12 @@
+#ifdef __CLING__
```

Comment:
We should also add a test with a `selection.xml` and a variant on the `linkdef` test with the `||` comment.

Reply:
Selection.xml unit test has been added. We deliberately ignore the `||` for the time being (unit test added).